src/service/standard-actions-impl.js

```diff
@@ -160,6 +198,9 @@ export class StandardActions {
           .catch((reason) => {
             dev().error(TAG, 'Failed to opt out of CID', reason);
           });
+      case 'toggleTheme':
```

Comment:
Still waiting on tests, which is the last necessary part to get this merged.

Reply:
Yes, I have added tests for the toggleTheme action. Unrelated question the usage of localStorage would still throw the eslint errors. So can I go ahead and add this file to allowlist in the forbidden-terms.js file?

Reply:
Add an `/*OK*/` in between `localStorage./*OK*/ getItem()`

Reply:
@jridgewell The `/*OK*/` flag does not seem to work with the localStorage so I have added this file to the allowlist in forbidden-terms.js file